recipes/msdf-atlas-gen/all/conanfile.py

```diff
@@ -68,4 +76,4 @@ def package_info(self):
 
         bin_path = os.path.join(self.package_folder, "bin")
         self.output.info("Appending PATH environment variable: {}".format(bin_path))
-        self.env_info.PATH.append(bin_path)
+        self.runenv_info.PATH.append(bin_path)
```

Comment:
This has a different syntax in v2, with this line the bin folder will be added to the [run environment](https://docs.conan.io/2/reference/tools/env/virtualrunenv.html), so to use the tool you'll have to activate the `conanrun.sh` and then it will be added to the PATH.
```suggestion
        self.runenv_info.prepend_path("PATH", bin_path)
```